### ISSUE TITLE
Profiling through code?

### ISSUE THREAD

**Issue by dyumanaditya** [pythonspeed/profila#14]
Hello,

Thank you for this profiler, I've been on the lookout for one that works with Numba!

Is it possible to profile specific function calls and not an entire file? I'm trying to profile a large python package that uses numba, but the part that uses numba cannot be called independently. Is there a way to do this currently?

This is what I get when I run the script that calls my package internally
```
/home/dyuman/miniconda3/envs/pyreason-dev/lib/python3.9/site-packages/numba/typed/typeddict.py:23: NumbaDebugInfoWarning: Could not find source for function: <function unicode_repr.<locals>.<lambda> at 0x7fffb27d0940>. Debug line information may be inaccurate.
  return dictobject._as_meminfo(dictobject.new_dict(keyty, valty,
/home/dyuman/miniconda3/envs/pyreason-dev/lib/python3.9/site-packages/numba/typed/typeddict.py:23: NumbaDebugInfoWarning: Could not find source for function: <function unicode_str.<locals>.<lambda> at 0x7fffb2bb1ee0>. Debug line information may be inaccurate.
  return dictobject._as_meminfo(dictobject.new_dict(keyty, valty,
/home/dyuman/miniconda3/envs/pyreason-dev/lib/python3.9/site-packages/pyreason/scripts/numba_wrapper/numba_types/interval_type.py:145: NumbaDebugInfoWarning: Could not find source for function: <function ctor.<locals>.impl at 0x7fffb2534d30>. Debug line information may be inaccurate.
  return Interval(np.float64(lower), np.float64(upper), static, np.float64(lower), np.float64(upper))
Filtering rules based on queries
Timestep: 0
Timestep: 1

Converged at time: 1
Fixed Point iterations: 2
**Total samples:** 4766 (99.7% non-Numba samples, 0.3% bad samples)
```

**Comment by itamarst** [comment]
What version gdb do you have installed? (`gdb --version`) You may have hit #12

**Comment by itamarst** [comment]
My guess is this will be fixed by release 0.3.0, please reopen if not.
